template-compose/app/src/main/res/values/styles.xml

```diff
@@ -1,8 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
-        <item name="colorPrimary">@color/greenCitrus</item>
-        <item name="colorPrimaryDark">@color/greenChristi</item>
-        <item name="colorAccent">@color/greenCitrus</item>
-    </style>
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar"/>
```

Comment:
Minor format issue
```suggestion
    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar" />
```

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/35fc2046d78d12e942c99cb679a0dfa618a1568d